app/views/gobierto_admin/gobierto_people/people/person_posts/index.html.erb

```diff
@@ -39,7 +39,9 @@
         <%= l(person_post.created_at, format: :short) %>
       </td>
       <td>
-        <%= person_post.tags.to_sentence %>
+        <% if person_post.tags.present? %>
+          <%= person_post.tags.to_sentence %>
```

Comment:
Alright, that smells like a premature optimization but I totally get your point. Let's leave the block 👍